grpclb/src/main/java/io/grpc/grpclb/GrpclbState.java

```diff
@@ -345,6 +361,7 @@ private void cancelLbRpcRetryTimer() {
   }
 
   void shutdown() {
+    logger.log(ChannelLogLevel.INFO, "[grpclb-<{0}>] Shutdown", serviceName);
```

Comment:
There is no test for this new log (which is INFO level)? I am assuming there are tests for logging because it is important in terms of debuggability?

Reply:
I would not bother test this. It's not critical. Also there is no much value verifying what's written here in the unit test.

Reply:
So unit tests were written for logging for a reason - to make sure a log is emitted when certain events happen even if verifying the exact message is not important (as you say). Just wanted to understand which is why I am asking for the issue link mentioned above.

Reply:
I think log being emitted for certain events is not worth a unit test. Because reading log is for a manual activity, we can always manually verify them and find if something need be added.

Reply:
SG. I thought there was an actual issue reported by a user so I asked.